Dockerfile

```diff
@@ -25,6 +25,6 @@ WORKDIR /tmp/
 RUN git clone https://github.com/perl-actions/ci-perl-tester-helpers.git --depth 1 && \
     cp ci-perl-tester-helpers/bin/* /usr/local/bin/ && \
     rm -rf ci-perl-tester-helpers && \
-    git config --global --add safe.directory '*'
+    git config --system --add safe.directory '*'
```

Comment:
why `local` would not be good enough?

Reply:
I'm not entirely sure what's happening here, but I get the impression that GitHub might be blowing away the `global` config. I think the `local` config would be local to a repository and since we don't have a checkout at this point, that shouldn't work. 

With the current containers `git config --list` comes up empty in GitHub CI, but it has the safe directory set when I deploy the image locally.

This also may not work, but it seems worth trying. 😄